### PR TITLE
[onkyo] Add support for TX-NR3007

### PR DESCRIPTION
**Pull request by peterhoeg**
I have an old TX-NR3007 which gets detected as "Unsupported", so this adds support.

It is not tested yet as it's currently building, but I will test locally and update this PR when things are working.

First foray into java - this will be fun!

### CODE REVIEW

File: bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="TX-NR3007">
+		<label>Onkyo TX-NR3007 AV Receiver</label>
+		<description>Network enabled Onkyo AV Receivers</description>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml]:
```suggestion
		<description>Network enabled Onkyo AV Receiver</description>
```

Comment (peterhoeg) [reply bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml]: All the Onkyo other models have ```<description>Network enabled Onkyo AV Receivers</description>``` (with the s).

I was assuming that it was used a group description and hence plural. 

```sh
$ grep '<description' tx*.xml | sort
tx-nr3007.xml:          <description>Network enabled Onkyo AV Receivers</description>
tx-nr414.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr474.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr509.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr515.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr525.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr535.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr545.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr555.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr575e.xml:          <description>Network enabled Onkyo AV Receivers</description>
tx-nr575.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr616.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr626.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr636.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr646.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr656.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr676.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr686.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr708.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr717.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr727.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr737.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr747.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr818.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr828.xml:           <description>Network enabled Onkyo AV Receivers</description>
tx-nr838.xml:           <description>Network enabled Onkyo AV Receivers</description>
```

In any case, shouldn't we make this the same as all the others? I will gladly submit a PR that changes them all to singular.

Comment (peterhoeg) [reply bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml]: Any thoughts @J-N-K ?

Comment (J-N-K) [reply bundles/org.openhab.binding.onkyo/src/main/resources/ESH-INF/thing/tx-nr3007.xml]: Sorry, didn‘t see your comment. They should all be the same. I would prefer the singular for all.

**Comment by peterhoeg** [comment]
All the descriptions have now been changed to singular.

**Comment by J-N-K** [comment]
Your commits are missing the sign-off. Can you fix that? Click on "details" in the DCO-check

**Comment by peterhoeg** [comment]
Done - sorry, wasn't aware of this requirement.

**Comment by J-N-K** [comment]
@kaikreuzer, did the Jenkins configuration change? We fail very often because the Map transformation feature is not available even though it is declared as dependency. I can‘t reproduce that locally and Travis also works.

**Comment by kaikreuzer** [comment]
> @kaikreuzer, did the Jenkins configuration change?

@pfink was the last one to do changes to the plans in order to fix the general build issues we had after introducing the super pom. Not sure if your issues here could be anyhow related...

**Pull request merged**